xmath/rand_test.go

```diff
@@ -94,3 +94,16 @@ func graphFill(rng xmath.RNG, degree int) float64 {
 	// Calculate p-value from chi-squared statistic.
 	return mathext.GammaIncRegComp(float64(degree*degree-1)/2, x/2)
 }
+
+var doNotOptimize uint64
```

Comment:
I don't belive an aggressive compiler would treat private vs public globals differently due to `go:linkname`, if the compiler would do single translation unit or LTO thing it could realise this is unused however it could do the same thing for a public global too.

Reply:
Good point. :) I tend to forget that the compiler does actually model around go:linkname.